src/baseApi.ts

```diff
@@ -27,19 +27,19 @@ export default abstract class BaseApi {
     };
   }
 
-  protected _get<T>(endpoint: string, options?: AxiosRequestConfig): Promise<AxiosResponse> {
+  protected _get<T>(endpoint: string, options?: AxiosRequestConfig) {
```

Comment:
Why remove the return types?

Reply:
The actual type returned and now inferred is `Promise<AxiosResponse<ApiResponse<T>>>` I believe. I deleted the explicit type as it removed some of the information.